### ISSUE TITLE
Built-In Git Server for Shaders

### ISSUE THREAD

**Issue by donran** [Pluppen/HealtyBoiChallenge#19]
I like to write code locally on my computer in Nano. Could we implement a git server for cloning and pushing each users shaders?

**Comment by Alexamakans** [comment]
This is high priority because copying and pasting is not ergonomic for many people.

**Comment by donran** [comment]
this is a high priority ticket

**Comment by Pluppen** [comment]
I agree that this is a high priority ticket, redirecting to the head developer on the Shaders team @Alexamakans could you give this ticket priority so that we can release it in the upcoming sprints

**Comment by Alexamakans** [comment]
@Pluppen Your comment has been duly noted. The issue has been scheduled for consideration in the upcoming Scrumbalooza Sprint Planning Poker duel.
